04-gas/gas-hero/gas-hero.html

```diff
@@ -16,8 +16,8 @@
       <div class="gas-hero-img-container">
         <img
           class="gas-hero-img"
-          src="../../00-overall/05-images/04-gas/gas-hero.webp"
-          alt="image gas hero"
+          src="../../00-overall/05-images/04-gas/gas-hero-house.png"
+          alt="an image of a house"
```

Comment:
please make alt more descriptive: alt="animated image of a big boiler on a house"

Reply:
ok